README.md

```diff
@@ -298,6 +354,97 @@ by its ID with the `custom.` prefix:
 />
 ```
 
+### Custom Queries within the catalog-info.yaml file of the Backstage Entity
+
+You can register custom queries in the `metadata.dynatrace.queries` property of
+a component within the `catalog-info.yaml` file
+
+```yaml
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: demo-backstage
+  description: Backstage Demo instance.
+  annotations:
+    backstage.io/kubernetes-id: kubernetescustom
+    dynatrace.com/guardian-tags: 'service=my-service,stage=development,novalue'
+  dynatrace:
+    queries:
+      - id: Error Logs
```

Comment:
When testing the functionality, I realised that the id is used as a name for the table. If there is no technical usage of this `id` property, it can stay as it is.